src/packages/sdk/src/IntrospectionEngine.ts

```diff
@@ -183,16 +183,11 @@ export class IntrospectionEngine {
       // eslint-disable-next-line no-async-promise-executor, @typescript-eslint/no-misused-promises
       async (resolve, reject): Promise<void> => {
         try {
-          const { PWD, ...env } = process.env
           const binaryPath = await resolveBinary('introspection-engine')
           debugRpc('starting introspection engine with binary: ' + binaryPath)
           this.child = spawn(binaryPath, {
             stdio: ['pipe', 'pipe', 'pipe'],
-            env: {
-              ...env,
-              // RUST_LOG: 'info',
-              // RUST_BACKTRACE: '1',
-            },
+            env: process.env,
```

Comment:
Looks unrelated to debug, does it fix something or it's a cleanup?

Reply:
This is cleanup